api/dynamodb/dynamodb.py

```diff
@@ -8,37 +9,43 @@ def __init__(self, dynamodb_table, settings: Settings):
         self.dynamodb_table = dynamodb_table
         self.settings = settings
 
-    def put_attraction(self, city: str, attraction: str):
+    def put_attraction(self, attraction: Attraction):
         self.dynamodb_table.put_item(
             Item={
-                "PK": city.title(),
-                "SK": attraction.title(),
+                "PK": attraction.city.title(),
+                "SK": attraction.name.title(),
             },
         )
 
-    def get_attraction(self, city, attraction):
+    def attraction_from_item(self, item: dict) -> Attraction:
```

Comment:
`attraction_from_item -> _attraction_from_item 
`
methods that are not meant to be used outside of the class should be prefixed with _ 
https://www.datacamp.com/tutorial/role-underscore-python
This doesn't change much - just good practice